components/Notification/NotificationItem.js

```diff
@@ -253,6 +253,32 @@ const NotificationItem = ({ notif, currentUser, notificationModal }) => {
 		)
 	}
 
+	if (notif.type === 'notification_content') {
```

Comment:
Tujuannya dicek juga hu 
`notif.to === currentUser`

Reply:
woke fixing hu